CHANGELOG.md

```diff
@@ -10,11 +10,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/).
 
 * The minimum supported Bazel version is now 0.27.
   `rules_haskell` supports Bazel up to 0.28.
-
   0.27 is a LTS release, which means upstream guarantees all new
   releases are backwards-compatible to it for 3 months. See the [Bazel
   Stability](https://blog.bazel.build/2019/06/06/Bazel-Semantic-Versioning.html)
   blog post for more information.
+* The repository name has changed, to follow the
+  new [Bazel rule guidelines][bazel-rule-guidelines]. It was
+  previously called `@io_tweag_rules_haskell`. It is now called
+  `@rules_haskell`. You should adapt your `WORKSPACE` file to match
+  this.
```

Comment:
I think this should also mention `--host_platform` in `.bazelrc`/`.bazelrc.local` explicitly. Many `grep` tools ignore dotfiles by default so it's easy to miss this and get confusing error messages.